meta-resin-common/recipes-connectivity/modemmanager/files/debug.diff

```diff
@@ -0,0 +1,11 @@
+--- a/data/ModemManager.service.in	2014-11-25 18:42:10.000000000 +0100
++++ b/data/ModemManager.service.in	2016-09-20 11:41:35.702860625 +0200
+@@ -5,7 +5,7 @@
+ [Service]
+ Type=dbus
+ BusName=org.freedesktop.ModemManager1
+-ExecStart=@sbindir@/ModemManager
++ExecStart=@sbindir@/ModemManager --debug
```

Comment:
Or you could use drop-ins to customize the service start. See https://www.freedesktop.org/software/systemd/man/systemd.unit.html for an explanation on drop-ins